src/Common/src/Microsoft/Win32/SafeHandles/SafeDirectoryHandle.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Win32.SafeHandles
```

Comment:
SafeHandles are all in the Microsoft.Win32 namespace...I should, however, add the .Unix suffix since it is Unix specific, similar to the others in that namespace